### PR TITLE
Add pyweed recipe

### PR DESCRIPTION
**Pull request by adam-iris**

### CODE REVIEW

File: recipes/pyweed/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [py2k]
```
Comment (synapticarbors) [inline recipes/pyweed/meta.yaml]: It looks like this project contains a python entry point. Please add it here:

https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#python-entry-points

File: recipes/pyweed/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    - setuptools
+  run:
+    - python
+    - obspy
+    - pyqt 4.11*
+    - qtconsole
+    - basemap
+    - pyproj
+    - pillow
+    - mock  # [py2k]
```
Comment (synapticarbors) [inline recipes/pyweed/meta.yaml]: is `mock` actually required to run the package?

File: recipes/pyweed/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+test:
```
Comment (synapticarbors) [inline recipes/pyweed/meta.yaml]: Typically we ask that you test all command line tools using:

https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#test-commands

but I'm not sure what the proper approach is given that this launches a GUI.

File: recipes/pyweed/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+  imports:
+    - pyweed
+
+about:
+  home: https://iris-edu.github.io/pyweed/
+  license: LGPLv3
```
Comment (synapticarbors) [inline recipes/pyweed/meta.yaml]: `license: LGPL-3.0`

https://opensource.org/licenses/alphabetical

File: recipes/pyweed/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+  license_file: LICENSE
+  summary: "PyWEED: A cross-platform app for retrieving event-based seismic data."
```
Comment (synapticarbors) [inline recipes/pyweed/meta.yaml]: add `dev_url: https://github.com/iris-edu/pyweed`

File: recipes/pyweed/meta.yaml (hunk 3 above)
Comment (adam-iris) [reply recipes/pyweed/meta.yaml]: I changed this to execute the unit tests

**Comment by synapticarbors** [comment]
@adam-iris -- Thanks for your contribution to conda-forge. A feedstock for this recipe should be generated soon and you'll be receiving several automated messages about it. When the feedstock has been rendered, please take a look at the README in that repo for information about updating and maintaining the recipe. One of the key points is all updates should be done by forking that repo and following a standard pull request workflow. If you haven't looked at it already the docs are quite helpful:

https://conda-forge.org/docs/

If you run into any difficulties please let us know.

**Pull request merged**